components/FeedList/FeedList.jsx

```diff
@@ -0,0 +1,53 @@
+import {useRoute} from '@react-navigation/native';
+import PropTypes from 'prop-types';
+import {FlatList, View} from 'react-native';
+import Photos from '../../screens/ProfileScreen/page/Photos/Photos';
+import SkeletonPhotos from '../Skeleton/SkeletonPhotos';
+import SkeletonPosts from '../Skeleton/SkeletonPosts';
+import Warning from '../Warning/Warning';
+import FeedItem from './FeedItem';
+import styles from './style';
+
+const FeedList = ({state, navigation, loading, showHide}) => {
+  const route = useRoute();
+  console.log(route.name);
+  const renderSwitchValue = (item, index) => {
+    if (route.name === 'Profile') {
+      if (showHide) {
+        return loading ? (
+          <SkeletonPosts />
+        ) : (
+          <FeedItem item={item} itemIndex={index} navigation={navigation} />
+        );
+      } else {
+        return loading ? <SkeletonPhotos /> : <Photos item={item} />;
+      }
+    } else {
+      return loading ? (
+        <SkeletonPosts />
+      ) : (
+        <FeedItem item={item} itemIndex={index} navigation={navigation} />
+      );
+    }
+  };
```

Comment:
I can't understand what is showHide, maybe because of the naming

Reply:
This variable have a Boolean value, which Mikael created for understanding and  display position of the SwitchSelector .

Reply:
If needed, I can change the name of this variable for a better explanation.